core/config.py

```diff
@@ -185,6 +185,8 @@ class Configuration(ConfigurationConstants):
 
     EXCLUDED_AUDIO_DATA_SOURCES = "excluded_audio_data_sources"
 
+    # In case an app version is not present, we can use this version as a fallback
+    # for all outgoing http requests with a custom user-agent
```

Comment:
Did you mean for all requests withOUT a custom ua here?

Reply:
Meant "default custom" UA, but it didn't come out right. "Without" is clearer.